### PR TITLE
Add Thanks / Recap post

### PR DESCRIPTION
**Pull request by ericholscher**
This is waiting on finishing Sketchnotes & Video uploads,
but we can get the content finished up.

The main thing here to review is the CoC section, which is probably worth a second set of eyes.

----
📚 Documentation preview 📚: https://writethedocs-www--2135.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/portland/2024/news/thanks-recap.rst [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+- Every participant agreed to the Code of Conduct in advance during registration.
+- The CoC, or a summary, was included on the conference website and around the venue.
+- The CoC was repeatedly mentioned in our conference introduction and announcements.
+- We specifically stressed that the CoC applies to all conference spaces including the social event.
+- We encouraged all attendees to report any incident, even if they were not sure whether it was a violation.
+
+We had 2 reports of CoC incidents at the conference:
+
+* An attendee reported an incident where someone shouted a comment during a talk about documentation tools that was not appropriate. We were unable to identify the person at the confernce, but staff apologized to the speaker, and noted during on-stage announcements and social media that this behavior was not acceptable.
```
Comment (janine-c) [inline docs/conf/portland/2024/news/thanks-recap.rst]:
```suggestion
* An attendee reported an incident where someone shouted a comment during a talk about documentation tools that was not appropriate. We were unable to identify the person at the conference, but staff apologized to the speaker, and noted during on-stage announcements and social media that this behavior was not acceptable.
```

File: docs/conf/portland/2024/news/thanks-recap.rst [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+* A staff member reported that an attendee was acting in an aggressive manner to staff about a conference selection process. This was addressed by the staff member with the attendee, and no further action was taken, other than the staff member noting the incident as a Code of Conduct report.
+
+Attendees can always report incidents, even after the conference,
+but emailing us at `conduct@writethedocs.org <mailto:conduct@writethedocs.org>`_.
```
Comment (janine-c) [inline docs/conf/portland/2024/news/thanks-recap.rst]:
```suggestion
by emailing us at `conduct@writethedocs.org <mailto:conduct@writethedocs.org>`_.
```

File: docs/conf/portland/2024/news/thanks-recap.rst [hunk 3]
```diff
@@ -0,0 +1,101 @@
+:template: {{year}}/generic.html
+:banner: _static/conf/images/headers/portland-2024-speakers.jpg
+:og:image: /_static/conf/images/headers/{{shortcode}}-{{year}}-opengraph.jpg
+
+.. post:: Apr 25, 2024
+   :tags: {{shortcode}}-{{year}}, thanks, recap
+
+Portland 2024 Recap - Talk Videos, Photos, and Community
+========================================================
```
Comment (plaindocs) [inline docs/conf/portland/2024/news/thanks-recap.rst]:
```suggestion
Portland 2024 Recap - Talk Videos, Photos, Sketchnotes, and Community
============================================================
```

File: docs/conf/portland/2024/news/thanks-recap.rst [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+Hey everyone,
+
+Our team is now emerging from the post-conference
```
Comment (plaindocs) [inline docs/conf/portland/2024/news/thanks-recap.rst]:
```suggestion
Our team is slowly emerging from the post-conference
```

File: docs/conf/portland/2024/news/thanks-recap.rst [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+Conference recap
+----------------
+
+At just under **350 attendees** this conference was another success.
+We had the usual mix of talks on the main stage along with Q&A,
+lightning talks, unconference sessions,
+writing sprints, social event, and sponsor expo.
+
+We used more spaces in the venue this year,
+and it really felt like the flow of the overall conference was improved.
```
Comment (plaindocs) [inline docs/conf/portland/2024/news/thanks-recap.rst]:
```suggestion
and it really felt like the overall flow of the conference was smoother.
```

File: docs/conf/portland/2024/news/thanks-recap.rst [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+Talk videos
+-----------
+
+`Videos of the talks`_ are now available on YouTube.
+This includes all conference talks with Q&A, as well as the lightning talks.
+You can subscribe to our `YouTube channel`_ so you don’t miss any future content.
```
Comment (plaindocs) [inline docs/conf/portland/2024/news/thanks-recap.rst]:
```suggestion
You can subscribe to our `YouTube channel`_ so you don’t miss any talks from our other conferences.
```

File: docs/conf/portland/2024/news/thanks-recap.rst [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+Thanks again to all our presenters, moderator, and folks who asked wonderful questions!
```
Comment (plaindocs) [inline docs/conf/portland/2024/news/thanks-recap.rst]: s/moderator/emcees/?

File: docs/conf/portland/2024/news/thanks-recap.rst [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+We have photos from the event available in our `Portland 2024 photo album <https://www.flickr.com/photos/writethedocs/albums/72177720316224454>`_, thanks to our wonderful photographer `Kay <https://twitter.com/goatlady>`_! 
+We really love the photos each year, and you're welcome to use them for non-commercial purposes, such as conference write ups, as per our `Creative Commons License <https://creativecommons.org/licenses/by-nc-sa/2.0/>`_.
+
+Sketchnotes
+-----------
+
+We also have sketchnotes for each of our talks!
+Check out our `Portland 2024 sketchnotes album on Flickr <https://www.flickr.com/photos/writethedocs/sets/72177720316395160>`_ to see the visual summaries of the talks and sessions.
+Thanks to our sketchnoter, `Dennis Dawson <https://www.linkedin.com/in/dennissdawson/>`_, for capturing the essence of the conference in such a unique way,
+while also presenting our first talk!
```
Comment (plaindocs) [inline docs/conf/portland/2024/news/thanks-recap.rst]:
```suggestion
Thanks to our sketchnoter, `Dennis Dawson <https://www.linkedin.com/in/dennissdawson/>`_, for capturing the essence of the conference in such a unique way, 
while also presenting the first conference talk on Monday, his first full length talk at Write the Docs!
```

**Review by plaindocs — APPROVED** [review]
Good stuff.

**Comment by jputrino** [comment]
LGTM!

**Pull request merged**